ehcache-impl/src/main/java/org/ehcache/impl/internal/executor/PooledExecutionService.java

```diff
@@ -82,7 +81,7 @@ private ThreadPoolExecutor getThreadPoolExecutor(String poolAlias) {
 
     poolAlias = poolAlias == null ? defaultPoolAlias : poolAlias;
     if (poolAlias == null) {
-      throw new IllegalArgumentException("Null pool alias provided and no default pool configured");
+      throw new IllegalArgumentException("Did you miss to configure default pool? Null pool alias provided and no default pool configured.");
```

Comment:
I'm wondering if we can phrase so that we don't start with a question... but also lead with a hint to the default pool? Maybe something like:
```java
throw new IllegalArgumentException("No default pool configured, and no explicit alias provided. Consider configuring a default pool.");
```